Documentation/articles/domain_driven_design/aggregate_root.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Aggregates and Aggregate Roots
+description: About Aggregates and Aggregate Roots
+keywords: domain-driven-design
+author: einari, smithmx
+---
+
+# Background
+
+An *Aggregate* is an object or collection of objects (Entities and Value Objects) that must be treated as a single unit for the
+purposes of a business transaction. The *Aggregate* will maintain all invariants for all objects that comprise the *Aggregate*,
+enforcing all business rules internal to the *Aggregate*.
+
+# Aggregate Root 
+
+All *Aggregates* have a single point of entry that is know as the root.  The *Aggregate Root* is the interface to the external world.
+All interaction with an *Aggregate* is via the *Aggregate Root*.  As such, an *Aggregate Root* **MUST** have a globally unique identifier
```

Comment:
Hey, just put "(GUID)" in brackets after globally unique identifier and that takes care of the comment above! :-)